<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>871</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 249, 211);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>45</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>85</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>15</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>15</number>
   </attribute>
   <row>
    <property name="text">
     <string>Neue Zeile</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Variable</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Values</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Min</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Max</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mean</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Std</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="idx_from_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Index  range</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idx_to_edit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_next">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_prev">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>output.pkl</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_update">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
  <widget class="QLabel" name="data_name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;PATH&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="lock_slide">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>280</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lock_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OVERLAY LOCK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_to">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_from">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="data_name_label2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_copy">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>650</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>copy to clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_copy_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>860</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>copy to clipboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="ACFlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Corr.</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fileEdit</tabstop>
  <tabstop>idx_from_edit</tabstop>
  <tabstop>idx_to_edit</tabstop>
  <tabstop>range_from</tabstop>
  <tabstop>range_to</tabstop>
  <tabstop>Btn_prev</tabstop>
  <tabstop>Btn_next</tabstop>
  <tabstop>table</tabstop>
  <tabstop>lock_slide</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
